cloud/kubernetes/README.md

```diff
@@ -25,11 +25,16 @@ kubectl run cockroachdb --image=cockroachdb/cockroach --restart=Never -- start-s
 ### Kubernetes version
 
 The minimum Kubernetes version to successfully run the examples in this
-directory without modification is `1.8`. If you want to run them on an older
-version of Kubernetes, use the files from the appropriate subdirectory (e.g. the
-`v1.7` directory for Kubernetes 1.7 or the `v1.6` directory for Kubernetes 1.6).
-Older Kubernetes versions that don't have their own directory are no longer
-supported.
+directory without modification is `1.18`.
+
+
+### Secure Mode
+
+The use of Kubernetes certificate authority to sign CockroachDBs certificates has been deprecated by Kubernetes. Due to 
+the deprecation to enable secure mode we currently only support the `bring-your-own-certs` configuration. 
+The deprecated method for using the Kubernetes signing authority is in the `v1.17` directory.
```

Comment:
Didn't you delete the v1.17 dir? Did you mean the kubernetes-signer dir?

Reply:
+1